### DISCUSSION TITLE
Fancy list transitions

### DISCUSSION THREAD

**Discussion by mihalikv** [bigskysoftware/htmx#1581] [Q&A]
I have no real need for these feature I am just curious if it's possible to achive similar behaviour as here https://v2.vuejs.org/v2/guide/transitions.html#List-Move-Transitions ? 

I mean, let's say that I have list of items and same list with same items with different order with come from server on click on button "shuffle".

**Comment by HeyHugo** [comment]
https://htmx.org/docs/#css_transitions
